src/Store/Features.re

```diff
@@ -448,6 +448,26 @@ let update =
       };
     let effect = [shEffect] |> Effect.batch;
     ({...state, signatureHelp}, effect);
+
+  | EditorCursorMove(editorID, _) =>
+    let maybeBuffer = Selectors.getActiveBuffer(state);
+    let maybeEditor =
+      state |> Selectors.getActiveEditorGroup |> Selectors.getActiveEditor;
```

Comment:
This might need to be merged up with the new `Feature_Layout` changes

Reply:
Just did this in e953d67! Let me know if it doesn't fit particularly well.